includes/plugins/class-newspack-newsletters.php

```diff
@@ -117,6 +123,45 @@ public static function get_metadata_key( $key ) {
 		return apply_filters( 'newspack_ras_metadata_key', $key, $prefix, $name );
 	}
 
+	/**
+	 * Normalizes contact metadata keys before syncing to ESP. If RAS is enabled, we should favor RAS metadata keys.
+	 * 
+	 * @param array $contact Contact data.
+	 * @return array Normalized contact data.
+	 */
+	public static function normalize_contact_data( $contact ) {
+		// Parse full name into first + last for MC, which stores these as separate merge fields.
+		if ( method_exists( 'Newspack_Newsletters', 'service_provider' ) && 'mailchimp' === \Newspack_Newsletters::service_provider() ) {
+			if ( isset( $contact['name'] ) ) {
+				$name_fragments                    = explode( ' ', $contact['name'], 2 );
```

Comment:
Hmm, do we need to normalize first + last name for Mailchimp for non-RAS sites, too?

Reply:
2a301c4c allows normalization for just first/last name to occur for Mailchimp, regardless of whether the site is using RAS.